_locales/en.json

```diff
@@ -522,5 +522,157 @@
+  "DC_STR_FILE": {
+    "message": "File",
+    "description": ""
+  },
+  "DC_STR_ENCRYPTEDMSG": {
+    "message": "Encrypted message",
+    "description": ""
+  },
+  "DC_STR_STATUSLINE": {
+    "message": "Sent with my Delta Chat Messenger: https://delta.chat",
+    "description": ""
+  },
+  "DC_STR_NEWGROUPDRAFT": {
+    "message": "Hello, I've just created the group \"%1$s\" for us.",
+    "description": ""
+  },
+  "DC_STR_MSGGRPNAME": {
+    "message": "Group name changed from \"%1$s\" to \"%2$s\".",
+    "description": ""
+  },
+  "DC_STR_MSGGRPIMGCHANGED": {
+    "message": "Group image changed.",
+    "description": ""
+  },
+  "DC_STR_MSGADDMEMBER": {
+    "message": "Member %1$s added.",
+    "description": ""
+  },
+  "DC_STR_MSGDELMEMBER": {
+    "message": "Member %1$s removed.",
+    "description": ""
+  },
+  "DC_STR_MSGGROUPLEFT": {
+    "message": "Group left.",
+    "description": ""
+  },
+  "DC_STR_SELFNOTINGRP": {
+    "message": "You must be a member of the group to perform this action.",
+    "description": ""
+  },
+  "DC_STR_NONETWORK": {
+    "message": "No network available.",
+    "description": ""
+  },
+  "DC_STR_E2E_AVAILABLE": {
+    "message": "End-to-end encryption available.",
+    "description": ""
+  },
+  "DC_STR_ENCR_TRANSP": {
+    "message": "Transport-encryption.",
+    "description": ""
+  },
+  "DC_STR_ENCR_NONE": {
+    "message": "No encryption.",
+    "description": ""
+  },
+  "DC_STR_FINGERPRINTS": {
+    "message": "Fingerprints",
+    "description": ""
+  },
+  "DC_STR_READRCPT": {
+    "message": "Return receipt",
+    "description": ""
+  },
+  "DC_STR_READRCPT_MAILBODY": {
+    "message": "This is a return receipt for the message \"%1$s\".",
+    "description": ""
+  },
+  "DC_STR_MSGGRPIMGDELETED": {
+    "message": "Group image deleted.",
+    "description": ""
+  },
+  "DC_STR_E2E_PREFERRED": {
+    "message": "End-to-end encryption preferred.",
+    "description": ""
+  },
+  "DC_STR_ARCHIVEDCHATS": {
+    "message": "Archived chats",
+    "description": ""
+  },
+  "DC_STR_STARREDMSGS": {
+    "message": "Starred messages",
+    "description": ""
+  },
+  "DC_STR_AC_SETUP_MSG_SUBJECT": {
+    "message": "Autocrypt Setup Message",
+    "description": ""
+  },
+  "DC_STR_AC_SETUP_MSG_BODY": {
+    "message": "This is the Autocrypt Setup Message used to transfer your key between clients.\n\nTo decrypt and use your key, open the message in an Autocrypt-compliant client and enter the setup code presented on the generating device.",
+    "description": ""
+  },
+  "DC_STR_SELFTALK_SUBTITLE": {
+    "message": "Messages I sent to myself",
+    "description": ""
+  },
+  "DC_STR_CANTDECRYPT_MSG_BODY": {
+    "message": "This message was encrypted for another setup.",
+    "description": ""
+  },
+  "DC_STR_CANNOT_LOGIN": {
+    "message": "Cannot login as %1$s.",
+    "description": ""
+  },
+  "DC_STR_SERVER_RESPONSE": {
+    "message": "Response from %1$s: %2$s",
+    "description": ""
+  }
```

Comment:
I'm leaving the descriptions empty for now until they have been documented in core.